### PR TITLE
Data cleaning, exploration and LSVI of HT31xx

### PR DESCRIPTION
**Pull request by anleyssen**

### CODE REVIEW

File: src/export_from_inboveg/HT31xx_data_preparation.Rmd [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+header_sel <- header %>%  
+  filter(str_detect(user_reference, this_sample))
+site_characteristics_sel <- site_characteristics %>% 
+  semi_join(header_sel, by = "recording_givid")
+vegetation_sel <- vegetation %>% 
+  semi_join(header_sel, by = "recording_givid")
+```
+
+ziet er ok uit
+
```
Comment (ToonHub) [inline src/export_from_inboveg/HT31xx_data_preparation.Rmd:390]:
```suggestion
doublerecords_site_characteristics <- site_characteristics %>%
  group_by(recording_givid, var, value) %>%
  mutate(n_records = n()) %>%
  ungroup() %>%
  filter(n_records > 1)
  
  doublerecords_vegetation <- vegetation %>%
  group_by(recording_givid, name, phenology_code) %>%
  mutate(n_records = n()) %>%
  ungroup() %>%
  filter(n_records > 1)
```

Comment (ToonHub) [reply src/export_from_inboveg/HT31xx_data_preparation.Rmd:390]: There are several double records for site_characterisics. You van check them with the code above. It is also the reason why you get a warning when using write_vc.

There also som some combinations of recording_givid, name, phenology_code with two cover codes. You noticed it in the data exploration, but you could also check it here.

Comment (ToonHub) [reply src/export_from_inboveg/HT31xx_data_preparation.Rmd:390]: If you press commit suggestion the suggested code is added in this brach, and you can run it in RStudio after you apply do a pull.

**Comment by ToonHub** [comment]
Some suggestions for the table site_caractheristics:

- NA's for var: can be added quite easily based on the var_code
- if you use de 'Voorwaarde'-name in the LSVI-database for 'var' you do not have to change it afterwards when calculating LSVI. I would defintely change the name '31xx_lsvi_horizontalestructuur' and '31xx_lsvi_boom' as it is not clear wath they mean. Maybe also remove '31xx' in the var names?
- For some values that are not numeric it is not clear wath they mean (for example booev, bozi,...). You can add a column 'value_description' or just change the value names.   
- I understand the use of is_above_LOQ for measuring depths. But I would not use is_below_LOQ in case of cover = '<1', because you also have cover values of 0, so you can quantify the cover below 1. I guess '<1' actually means that the cover is > 0 and < 1. You might use 0.5 for 'value_numeric' (as you do later in the lsvi-calculation) and use '<1' for 'value'

**Comment by ToonHub** [comment]
In the LSVI-calculation you use '1-out-all-out' for the aggregation. If you want to use the method that was used for the HR-reporting, you sould use Aggregatiemethode = "RapportageHR".

**Comment by ToonHub** [comment]
To make the code better readable I suggest to always start a new line after %>%

**Comment by anleyssen** [comment]
Suggested changes in HT31xx data preparation are made: 

- add value_description for class variables, 
- change 0-x-1 and <1 into 0.5 for value_numeric and change is_below_LOQ, 
- replace NA in var and value_description

Some minor changes in HT31xx_LSVI (text edits) are also made. 
Can @ElsLommelen also review the code for LSVI-calculation?

**Comment by anleyssen** [comment]
The suggested changes of Els Lommelen are made in HT31xx LSVI. 
Other 'to do's' can't be solved at the moment; so @ToonHub, you can merge this branch with the master.

**Review by ToonHub — APPROVED** [review]
OK! The code runs fine.

**Pull request merged**